README.Rmd

```diff
@@ -159,3 +159,39 @@ By contributing to this project, you agree to abide by its terms.
 ```{r message=FALSE, warning=FALSE}
 citation("simulist")
 ```
+
+## Related projects
+
+This project has some overlap with other R packages. Here we list these packages and provide a table of features and attributes that are present for each package to help decide which package is appropriate for each use-case.
+
+In some cases the packages are dedicated to simulating line list and other epidemiological data (e.g. {simulist}), in others the line list simulation is one part of a wider R package (e.g. {EpiNow}). 
+
+- [`{LLsim}`](https://github.com/jrcpulliam/LLsim) simulates line list data using a stochastic SIR model with a fixed population with observation and reporting delays.
+- [`{simulacr}`](https://github.com/reconhub/simulacr) uses a branching process model to simulate cases and contacts for an outbreak. It simulates transmission of infections using other epidemiological R packages (`{epicontacts}` and `{distcrete}`) to parameterise and plot simulated data.
```

Comment:
so this package does the same as {simulist}? I think it'd be good to add a sentence to each of these highlighting what simulist brings to the table, for example user friendliness or the package being self contained and not relying on other packages- obviously in a subtle way without being negative about the other available packages

Reply:
I'm not sure that going into detail on the differences between the packages is required for the `README`. This section is more of a guide to showcase which other packages are available with a brief description. In my opinion it is up to the user to decide what is the appropriate tool for their use case. 

As you mentioned, it's difficult to advocate for {simulist} versus the other listed packages without being negative about them. 

Hopefully the table of features will provide a high-level guide to which tool is appropriate to use (e.g. showing which packages are no longer maintained).